testsuite/WIKI.md

```diff
@@ -154,9 +146,6 @@ The simulation data used in tests are all released under the same license as MDA
 * contributions from developers and users
 * Protein Databank
 
-## Purposely failing test ##
-
-It is sometime convenient to have a failing test. This is helpful, for instance, when testing a plugin for nosetests, or to make sure a script catches failure as expected. The `test_failure` test in `test_failure.py` can be made to fail by setting the `MDA_FAILURE_TEST` environment variable.
```

Comment:
This can now be done with `--xfail` i think. You can leave the information here but please check the pytest flag